src/dmd/attrib.d

```diff
@@ -929,6 +928,7 @@ extern (C++) final class StaticIfDeclaration : ConditionalDeclaration
 {
     ScopeDsymbol scopesym;
     bool addisdone;
+    bool onStack = false;
```

Comment:
It took me a little while to understand what "onStack" means, but I see now.

Reply:
Added a comment.